app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split(" ")
```

Comment:
command_ls is bad naming. Also, your code will raise an Error in two cases
1. command == 'cp file1.txt'
2. command == 'cp file1.txt file2.txt file3.txt'